airflow/providers/apprise/hooks/apprise.py

```diff
@@ -71,7 +72,7 @@ def notify(
         title: str | None = None,
         notify_type: NotifyType = NotifyType.INFO,
         body_format: NotifyFormat = NotifyFormat.TEXT,
-        tag: str | Iterable[str] | None = None,
+        tag: str | Iterable[str] = MATCH_ALL_TAG,
```

Comment:
is this braking change?

Reply:
Yes! Passing None will not send the notification at all. Our team struggled for 2 hours to find this bug.

Reply:
How it should work according to the Apprise:

>         if the tag is specified (either a string or a set/list/tuple
>         of strings), then only the notifications flagged with that
>         tagged value are notified.  By default, all added services
>         are notified (tag=MATCH_ALL_TAG)